pkgs/by-name/sw/swhkd/package.nix

```diff
@@ -0,0 +1,68 @@
+{ lib
+, rustPlatform
+, fetchurl
+, fetchFromGitHub
+, pkg-config
+, scdoc
+, libgcc
+, systemd
+}:
+rustPlatform.buildRustPackage rec {
+    pname = "swhkd";
+    version = "1.2.1-unstable-2024-04-06";
+
+    # split-output derivation, since there's a fair amount of associated data for
+    # pkexec and such.
+    outputs = [ "bin" "man" "out" ];
```

Comment:
but don't we need always all that data? Then it would be easier to keep everything in out.

Reply:
considering the previous PR got two approvals while not generating any of that data, it would seem not.

Reply:
True but we require quite some extra code because of the split outputs and removing them would make the module quite a bit simpler and we don't loose much.